app/src/main/java/com/sdp/swiftwallet/presentation/main/fragments/MessageFragment.java

```diff
@@ -1,25 +1,143 @@
 package com.sdp.swiftwallet.presentation.main.fragments;
 
+import static android.content.ContentValues.TAG;
+
 import android.os.Bundle;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
 
 import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.SetOptions;
 import com.sdp.cryptowalletapp.R;
+import com.sdp.swiftwallet.common.FirebaseUtil;
+import com.sdp.swiftwallet.presentation.friend.fragments.FriendItem;
+import com.sdp.swiftwallet.presentation.friend.fragments.FriendItemFragment;
+import com.sdp.swiftwallet.presentation.friend.fragments.FriendItemRecyclerViewAdapter;
+import com.sdp.swiftwallet.presentation.wallet.fragments.WalletItemFragment;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+
+public class MessageFragment extends Fragment{
+    private FirebaseAuth mAuth;
```

Comment:
You should be able to use hilt injection for this instance, at least for the next weeks